plotly/plotlyfig_aux/handlegraphics/updateScatter.m

```diff
@@ -122,6 +122,24 @@ function updateScatter(obj,scatterIndex)
     
     %---------------------------------------------------------------------%
     
+    %-scatter z-%
+    if ~verLessThan('matlab','8.4') 
```

Comment:
Thanks, that's a much better solution... hopefully feature('UseHG2') continues to exist in future versions!